### DISCUSSION TITLE
Remove compat versions

### DISCUSSION THREAD

**Discussion by Drevoed** [zerobias/effector#712] [Ideas]
IE will reach its EOL on June 15th 2022. My proposal is to remove compat versions when this will happen. This will also allow us to use ES6 APIs, such as ([,Weak]Sets/Maps). And to further improve performance as well.

**Comment by igorkamyshev** [comment]
Many developers use `compat` versions for Smart TVs as well. Furthermore, they cannot control environment, so after this update their application will stick with the current version of Effector without possibility to update. It means they won't get any bugfixes, security updates, etc.

I think, it is terrible DX, so, in my opinion, we should not drop `compat` versions.

**Comment by zerobias** [comment]
No depreciation of compatibility packages, this is not an option. Why are you not suggested to raise minimal version supported version?

Map and Set actively used for a long time and there is no need to use anything weak, so I see there no viable motivation at the first place
